ui/app/models/node.js

```diff
@@ -9,17 +9,16 @@ import { alias, and, equal } from '@ember/object/computed';
 export default Model.extend({
   name: attr('string'),
   // https://developer.hashicorp.com/vault/api-docs/system/health
-  initialized: attr('boolean'),
-  sealed: attr('boolean'),
-  isSealed: alias('sealed'),
   standby: attr('boolean'),
   isActive: equal('standby', false),
-  clusterName: attr('string'),
```

Comment:
clusterName isn't used anywhere, so removed it from the model. The other values are also available on `seal-status` which will be available in a chroot scenario, so I moved those below to that section